modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -15,6 +15,20 @@ As a network administrator, you can configure a network attachment definition of
 .Procedure
 
 . Create a network attachment definition manifest.
++
+[NOTE]
+====
+The network attachment definition must be in the same namespace as the pod or virtual machine.
```

Comment:
It might be more elegant to say "Create a network attachment definition in the same namespace as the virtual machine."